### PR TITLE
chore: add CONTRIBUTING.md page to Dialtone Icons

### PR DESCRIPTION
**Pull request by ninamarina**
# chore: add CONTRIBUTING.md page to Dialtone Icons

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1483

## :book: Description

Adds a contributing page to dialtone-icons inside the monorepo. It's base off the previous contributing page.

### CODE REVIEW

**Review by iropolo — APPROVED** [review]
Looks, Ok, wait for a design review @yorbi-dp @fede-dp , Thanks!

**Review by braddialpad — APPROVED** [review]
Looks fine to me though a lot of this stuff is repeated in multiple documents (roles, commit convention, etc) Wondering if we should move this stuff to the parent document. Anyways, that's outside the scope of this PR, this looks good.

**Review by yorbi-dp — COMMENTED** [review]
Nice we can include the contribution info for this packages! Thanks Nina! 😄

File: packages/dialtone-icons/.github/CONTRIBUTING.md
Comment (yorbi-dp) [inline packages/dialtone-icons/.github/CONTRIBUTING.md]: It's `keywords.json` the file with the categories and keywords.

File: packages/dialtone-icons/.github/CONTRIBUTING.md
Comment (yorbi-dp) [inline packages/dialtone-icons/.github/CONTRIBUTING.md:89]: Should we add the step-by-step to add an icon in the package?
- In the [Icons Figma file](https://www.figma.com/file/zz40wi0uW9MvaJ5RuhcRZR/DT9-Icon-Library?type=design&node-id=10023-2864&mode=design&t=zfm4YTgkSdohnCrw-4) select and export each icon as SVG. 
- Place the exported SVG file(s) in the appropriate folder category inside `./src/svg/`
- Run `pnpm run build` adding the name and keywords in the `keywords.json` file.
- Add the keywords array as the value. e.g.
```
{
   "arrows": {
     "arrow-left": ["arrow", "left", "direction"],
     ...
   }
}
``` 
- Add, commit and push your changes as mentioned from step 4 in [Making a pull request](#making-a-pull-request).
- Open a pull request.
- Once approved it can be merged into main and will go out in the next `dialtone` release.

[Learn more on How to craft an icon.](https://dialtone.dialpad.com/design/icons/#crafting-an-icon)

Not sure where should this be?

File: packages/dialtone-icons/.github/CONTRIBUTING.md
Comment (yorbi-dp) [inline packages/dialtone-icons/.github/CONTRIBUTING.md]: We can add something like:
[Check the Icon catalog](https://dialtone.dialpad.com/design/icons/)

**Comment by ninamarina** [comment]
> Looks fine to me though a lot of this stuff is repeated in multiple documents (roles, commit convention, etc) Wondering if we should move this stuff to the parent document. Anyways, that's outside the scope of this PR, this looks good.

I agree all this page could be summarized. Probably the most important section is this one: https://github.com/dialpad/dialtone/pull/105/files#diff-2e26192c13c3c1ef3e307cf5fe07dbff0f1c9ad42ea0ca2d3e0e1e7e509c5158R65
And all the other info is more generic to the repo

**Review by yorbi-dp — CHANGES_REQUESTED** [review]

File: packages/dialtone-icons/.github/CONTRIBUTING.md
Comment (ninamarina) [reply packages/dialtone-icons/.github/CONTRIBUTING.md:89]: Added this info in a new section "Adding an icon"

**Review by yorbi-dp — APPROVED** [review]
LGTM! Thank you Nina! 🙌

**Pull request merged**
